bam_consensus.c

```diff
@@ -1483,15 +1486,16 @@ int calculate_consensus_gap5(hts_pos_t pos, int flags, int depth,
 
     int d1 = 0, d2 = 0;
     double nd1 = 0, nd2 = 0;
-    for (int i = 0; i < 100; i++) {
-        if (!poly_dist[0][i] && !poly_dist[1][i])
+    int k;
+    for (k = 0; k < 100; k++) {
+        if (!poly_dist[0][k] && !poly_dist[1][k])
             continue;
 
-//        fprintf(stdout, "%ld %d %2d %2d\n", pos, i, poly_dist[0][i], poly_dist[1][i]);
-        d1 += (i+1)*poly_dist[0][i];
-        d2 += (i+1)*poly_dist[1][i];
-        nd1 += poly_dist[0][i];
-        nd2 += poly_dist[1][i];
+//        fprintf(stdout, "%ld %d %2d %2d\n", pos, k, poly_dist[0][k], poly_dist[1][k]);
+        d1 += (i+1)*poly_dist[0][k];
+        d2 += (i+1)*poly_dist[1][k];
```

Comment:
`i` was on this line twice: `i+1` and `[i]`.  I don't think the variable needed renaming though as I can't see any shadowing.